### PR TITLE
Convert everything to database

### PR DESCRIPTION
**Pull request by SketchMaster2001**
As this is a quite huge change, I made this into a PR so we can review it. @spotlightishere I am looking mainly for your input. As the title says, everything is now handled by Postgres. Restaurants will have their own ID which is the primary key, same goes for menu and food items. Some point in the future I would like to make a web panel for easy restaurant and food item creation.

### CODE REVIEW

**Review by spotlightishere — APPROVED** [review]
Looks okay to me! I have a few comments but otherwise life looks good - we might want to resolve these out of this PR as well

File: food.py [hunk 1]
```diff
@@ -105,11 +105,7 @@ def error_api():
         return exceptions.NotFound()
 
 
-@app.route("/itemimg/<filename>")
+@app.route("/itemimg/1/<filename>")
```
[the remark this answers is not in the document]
Comment (SketchMaster2001) [reply food.py:108]: This is derived from the category code, should fix

File: responses.py [hunk 2]
```diff
@@ -26,80 +30,95 @@ def basket_reset(request):
...
+                                    "menuCode": 1,
+                                    "itemCode": 1,
+                                    "image": 1,
+                                    "isSoldout": 0,
+                                    "info": "Known to blow up in your face",
+                                    "price": "5.99",
+                                }
+                            },
+                        }
+                    },
```
[the remark this answers is not in the document]
Comment (SketchMaster2001) [reply responses.py:80]: This is used just so the cart will load, we can find a different description

File: models.py [hunk 3]
```diff
@@ -1,11 +1,53 @@
...
+    def process_result_value(self, value, dialect):
+        if value is not None:
+            value = json.loads(value)
+        return value
 
 
 class Shops(db.Model):
-    name = db.Column(db.String, nullable=False, primary_key=True)
+    name = db.Column(db.String, nullable=False)
     category_code = db.Column(db.Integer, nullable=False)
```
[the remark this answers is not in the document]
Comment (SketchMaster2001) [reply models.py]: Fixed as of 8182a6a

File: responses.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (SketchMaster2001) [reply responses.py]: Changed as of 89a150b

File: food.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (SketchMaster2001) [reply food.py]: Fixed as of 1cc7e79

**Review by spotlightishere — APPROVED** [review]
Looks great, thank you!

**Pull request merged**